docs/developers/strange_problems.md

```diff
@@ -0,0 +1,33 @@
+# Something strange has gone wrong
+
```

Comment:
😆 🤣

Reply:
Yeah this was the best I could come up with 🫠 Do you think it's okay like this, or should I try to come up with a different name?